news/2016/2016-01-04-osuweekly-42.md

```diff
@@ -14,7 +14,7 @@ I could barely catch a break with how action packed this week was. A certain leg
 
 **[There's a new community poll to kick-start 2016](https://osu.ppy.sh/community/forums/topics/404210)**! The osu! community is now bigger than ever before. In the past, people were drawn to the secrets of circle clicking through word of mouth and peeking over a friend's shoulder. Today, many [prominent figures](https://osu.ppy.sh/home/news/2015-10-19-osuweekly-32) and sources helped grow the community into what it is now. How did you hear about the game? Let us know in the thread!
 
-There is no need for introductions. If I were to write a whole article on just how many noteworthy accomplishments [Cookiezi](https://osu.ppy.sh/users/124493) has achieved since he joined the game, I would surely run my digital ink dry. Thus, we have but the most recent today. [Blue Zenith [Four Dimensions]](https://osu.ppy.sh/beatmaps/658127) has a new top score: a Hard Rock play with an unbelievable accuracy of 99.71%! This completely shatters the previous top performance of osu! standard, and marks the first time any single play surpassed the 700pp mark.   
+There is no need for introductions. If I were to write a whole article on just how many noteworthy accomplishments [Cookiezi](https://osu.ppy.sh/users/124493) has achieved since he joined the game, I would surely run my digital ink dry. Thus, we have but the most recent today. [Blue Zenith [Four Dimensions]](https://osu.ppy.sh/beatmaps/658127) has a new top score: a Hard Rock play with an unbelievable accuracy of 99.71%! This completely shatters the previous top performance of osu! standard, and marks the first time any single play surpassed the 700pp mark.
```

Comment:
```suggestion
There is no need for introductions. If I were to write a whole article on just how many noteworthy accomplishments [Cookiezi](https://osu.ppy.sh/users/124493) has achieved since he joined the game, I would surely run my digital ink dry. Thus, we have but the most recent today. [Blue Zenith [Four Dimensions]](https://osu.ppy.sh/beatmapsets/292301#osu/658127) has a new top score: a Hard Rock play with an unbelievable accuracy of 99.71%! This completely shatters the previous top performance of osu! standard, and marks the first time any single play surpassed the 700pp mark.
```